### ISSUE TITLE
Parsing seems to skip arrays

### ISSUE THREAD

**Issue by kevinkleine** [polyforest/jsdoc-tsimport-plugin#2]
Your plugin is a life saver for me. Thank you. I noticed that it doesn't seem to parse imports that are embedded in arrays, such as:

```
/**
 * @method
 * @param {Object} someObject
 * @returns {Array<import("../path/file.js").Declaration>}
 */
```

Am I doing something wrong?

**Comment by nbilyk** [comment]
Glad to hear it!  
Sorry for the slow response, I'll fix this now.

**Comment by nbilyk** [comment]
Fixed in v1.0.2

Thanks for trying it out

**Comment by nbilyk** [comment]
![image](https://user-images.githubusercontent.com/589193/106373550-4add3580-6340-11eb-9b80-3be9ef0cb4bd.png)
